node/pkg/governor/governor_prices.go

```diff
@@ -309,7 +309,7 @@ func CheckQuery(logger *zap.Logger) error {
 	logger.Info("Instantiating governor.")
 	ctx := context.Background()
 	var db db.MockGovernorDB
-	gov := NewChainGovernor(logger, &db, common.MainNet)
+	gov := NewChainGovernor(logger, &db, common.MainNet, false)
```

Comment:
Yes, thanks!

Reply:
Just taking a look at this more closely. I don't think the check query test function interacts with flow cancelling at all. The flow cancel token list doesn't store price information, only the OriginChain, OriginAddress, and Symbol.

Reply:
What I meant is that if the flow cancel token list was bad (it returns an error if `vaa.StringToAddress` fails), passing in true above would cause `check_query` to fail, allowing the problem to be caught earlier.

Reply:
Ah I see. Sure, no harm adding it